docs/design_decisions.md

```diff
@@ -1,36 +1,40 @@
 # Design decisions HaalCentraal API's
-Dit document beschrijft ontwerpkeuzes die gemaakt zijn voor het ontwerpen en specificeren van de API's binnen het programma HaalCentraal 
+Dit document beschrijft ontwerpkeuzes die gemaakt zijn voor het ontwerpen en specificeren van de API's binnen het programma HaalCentraal
 
-## Naamgevingsconventie 
+## Naamgevingsconventie
 
-Naamgevingsconventie is altijd zo duidelijk mogelijk te benoemen wat iets is. 
+Naamgevingsconventie is altijd zo duidelijk mogelijk te benoemen wat iets is.
+Hoofdregel is altijd:
+1. propertynamen moeten zoveel mogelijk zelfverklarend zijn (lezen van de description om de betekenis te begrijpen is liefst niet nodig)
+2. propertynamen zijn zo kort als mogelijk om toch voldoende duidelijk en onderscheidend te zijn en niet langer dan daarvoor nodig
 
-* Relaties: De naam van de relatie plus de naam van de gerelateerde resource, tenzij er een reden is om dat niet te doen.
+Hieronder staat een aantal algemene richtlijnen voor naamgeving van properties.
+* Relaties (links naar gerelateerde resources): De naam van de relatie plus de naam van de gerelateerde resource.
   * Bijvoorbeeld:
 
   ligt in + woonplaats = ligtInWoonplaats
-  
+
```

Comment:
Dit wilde je toch uit de richtlijn halen. Volgens mij hebben we op dit punt gisteren lang gediscussieerd (Neem je de relatie op in de property-naam van relaties. Volgens mij was de conclusie dat je de property waarin je de identificatie van een gerelateerde resource opneemt alleen de naam van de gerelateerde resource geeft, tenzij er meerdere relaties naar hetzelfde object zijn. Nu wordt in de voorbeelden alleen maar het opnemen van de relatienaam getoond.